app/policies/school_policy.rb

```diff
@@ -26,4 +30,14 @@ def support_third_line_editable?
   def update_computacenter_reference?
     user.is_computacenter?
   end
+
+private
+
+  def devolved_management_school_user?
+    record.orders_managed_by_school? && user.school_ids.include?(record.id)
+  end
+
+  def responsible_body_user?
+    user.responsible_body_id.present? && user.responsible_body_id == record.responsible_body_id
```

Comment:
Should this also check on the user object if orders_devices = true? At least for the devices_orderable? context on line 12. If so we also need to rename this method.

Reply:
See my reply above